accounting/docs.md

```diff
@@ -102,3 +102,26 @@ Circular receipts record instances where we have paid one of our own invoices.
 - TxID: The payment hash of the invoice.
 - Reference: The preimage of the invoice.
 - Note: Optionally set if the invoice had a memo attached, was overpaid, or was a keysend.
+
+### Forwards
+Forward represents a forward through or node. While these entries do not change our balance except for fees earned, we include them for completeness. 
```

Comment:
nit: I'd phrase the beginning to be a bit more specific. Something like: "A forward represents/is a forwarded payment that passes trough one of our channels." (could be wrong though)

Reply:
Definitely better to be more specific, that's what docs are for after all.